tests/kas_with_instance.spec.ts

```diff
@@ -177,3 +193,43 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
+
+// test_6acl.py test_kafka_create_consumer_group_and_check_dashboard
+test('create consumer group and check dashboard', async ({ page }) => {
+  const instanceLinkSelector = page.getByText(testInstanceName);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await waitForKafkaReady(page, testInstanceName);
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Connection').click();
+
+  const broker = await getBootstrapUrl(page, testInstanceName);
+  console.log('broker: ' + broker);
```

Comment:
you should rename it to `bootstrapUrl` for better readability

Reply:
ok I rename it to `bootstrapUrl`

Reply:
Can we resolve this conversation @Frawless ?